### ISSUE TITLE
processe killed 

### ISSUE THREAD

**Issue by 0xmid00** [Lupin-Holmes/sandwich#4]
./sandwich.rb 5ca7c2fa-4f85-11ef-b335-cf47ff697e01 b7bc913e-54ee-11ef-b335-cf47ff697e01 -o uuid.txt
[1]    49859 killed     ./sandwich.rb 5ca7c2fa-4f85-11ef-b335-cf47ff697e01  -o uuid.txt

**Comment by ELHart05** [comment]
yeah true that happens when you got a lot of uuids to generate, performance issues xd, the solution for that is to parallelize the work (use google collab for example or a performant vm)
